DESCRIPTION

```diff
@@ -35,3 +36,5 @@ Language: en-GB
 LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
+Remotes:  
+    mllg/checkmate
```

Comment:
Shall we use remotes other than CRAN?

Reply:
Ah, already moved it to Suggests, where it belongs.

Reply:
It was probably meant to be added to "Suggests" section